api/occupancy/OccupancyDao.php

```diff
@@ -14,8 +14,9 @@ class OccupancyDao
     public static function processFeedback($feedback, $epochFeedback)
     {
         date_default_timezone_set('Europe/Brussels');
+        $dateParameter = substr($feedback['date'], -2) . substr($feedback['date'], -4, 2) . substr($feedback['date'], -6, 2);
```

Comment:
That's when you trust the date parameter is strictly as given. We should really parse dates according to the ISO8601 standard, and not according to our own proprietary way of doing it. Let's put this as a issue